src/opendr/control/multi_object_search/requirements_installations.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020-2023 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import igibson
+import shutil
+import os
+from opendr.engine.constants import OPENDR_SERVER_URL
+from urllib.request import urlretrieve
+from igibson.utils.assets_utils import download_ig_dataset
+from pathlib import Path
+
+TEMP_SAVE_DIR = Path(__file__).parent / "multi_object_search_tmp"
+
+
+"""
+This file is specifically for downloading the iGibson dataset (without the key)
+and the inflated maps
+"""
+
+
+class multi_object_search_requirements():
+    def __init__(self):
+        self.temp_path = TEMP_SAVE_DIR
+
+    def download_dataset(
+            self,
+            path=None,
+            url=OPENDR_SERVER_URL +
+            "control/multi_object_search/"):
+
+        path = self.temp_path
+        print("-----> Start downloading iGibson assets")
+        download_ig_dataset()
+        print("-----> Successfully downloaded iGibson Dataset")
+        inflated_files = [
+            "Beechwood_0_int", "Beechwood_1_int", "Benevolence_0_int",
+            "Benevolence_1_int", "Benevolence_2_int", "Ihlen_0_int",
+            "Ihlen_1_int", "Merom_0_int", "Merom_1_int", "Pomaria_0_int",
+            "Pomaria_1_int", "Pomaria_2_int", "Rs_int", "Wainscott_0_int",
+            "Wainscott_1_int"]
+        print("-----> Start Download iGibson prerequisits")
```

Comment:
```suggestion
        print("-----> Start Download iGibson prerequisites")
```

Reply:
I remember fixing the issues but now sure why they got lost. Still, git it giving me rejections due to tip of my branch is behind although I pulled everything. I am not sure if that is correct now. So if you could please take a look ?
Thnaks